sql/core/src/test/resources/tpcds-query-results/v1_4/q41.sql.out

```diff
@@ -0,0 +1,6 @@
+-- Automatically generated by TPCDSQueryTestSuite
+
+-- !query schema
+struct<i_product_name:string>
+-- !query output
+
```

Comment:
This should be:
```
ableationableought                                
anticallyeingese                                  
callycallyeingese                                 
oughtationableought                               
```
A PR to fix this is: https://github.com/apache/spark/pull/31940